src/test/rustdoc/extern-default-method.rs

```diff
@@ -1,9 +1,21 @@
 // aux-build:rustdoc-extern-default-method.rs
 // ignore-cross-compile
+// ignore-tidy-linelength
```

Comment:
Tidy does not complain about the overly long (> 100 characters) lines 18 and 20 (the ones with `@has` and `@!has`) but it does with line 19 which is 101 characters longs :confused:. Instead of awkwardly wrapping the `@count` line with `\`, I decided to instead silence tidy here.

Reply:
Don't worry, it's completely fine!